pom.xml

```diff
@@ -140,7 +146,6 @@
                     <source>8</source>
                     <doclint>-html</doclint>
                     <quiet>true</quiet>
-                    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
What's this for?

Reply:
Maven is complaining about its inclusion